editor/__main__.py

```diff
@@ -24,11 +24,11 @@
     file_names = [os.path.basename(file.name) for file in args.files]
     for file in args.files:
         file.close()
-    if not file_names:
-        parser.error("No files provided.")
 else:
-    if len(configs) != 1:
-        parser.error("Unable to resolve okpy configs, files to be tested must be specified explicitly.")
-    with open(configs[0]) as f:
-        file_names = json.loads(f.read())["src"]
-local_server.start(file_names, args.port)
+    file_names = []
+    if len(configs) > 1:
+        parser.error("Multiple okpy configs detected, files to be tested must be specified explicitly.")
+    elif len(configs) > 0:
+        with open(configs[0]) as f:
+            file_names = json.loads(f.read())["src"]
+local_server.start(file_names, args.port, args.nobrowser)
```

Comment:
this looks very confusing. Make `nobrowser` store true and call it with `not args.nobrowser`